docs/docusaurus/versioned_docs/version-0.17/changelog.md

```diff
@@ -2565,7 +2565,7 @@ title: Changelog
 * [MAINTENANCE] Provide better error messaging if batch_request is not supplied to DataAssistant.run() ([#5473](https://github.com/great-expectations/great_expectations/pull/5473))
 * [MAINTENANCE] Adds run time envvar for Snowflake Partner ID ([#5485](https://github.com/great-expectations/great_expectations/pull/5485))
 * [MAINTENANCE] fixed algolia search page ([#5099](https://github.com/great-expectations/great_expectations/pull/5099))
-* [MAINTENANCE] Remove pyspark<3.0.0 constraint for python 3.7 ([#5496](https://github.com/great-expectations/great_expectations/pull/5496))
+* [MAINTENANCE] Remove pyspark 3.0.0 constraint for python 3.7 ([#5496](https://github.com/great-expectations/great_expectations/pull/5496))
```

Comment:
I didn't look through this extensively (or pretty much at all) but happened to see this change. Looks incorrect. I think you want to replace the `<` with the corresponding html code. I think either `&lt;` or `&#60;` would work, but you'd want to double check that. I'm guessing there are other examples of this in this PR.